tests/BrainSentimentIndicatorUniverseTests.cs

```diff
@@ -32,6 +32,30 @@ namespace QuantConnect.DataLibrary.Tests
     [TestFixture]
     public class BrainSentimentIndicatorUniverseTests
     {
+        [Test]
+        public void ReaderTest()
+        {
+            var factory = new BrainSentimentIndicatorUniverse();
+            var line = "AAPL R735QTJ8XC9X,AAPL,869,516,0.1196,,,5101,3176,0.0976,-0.169,0.0888";
+
+            var config = new SubscriptionDataConfig(
+                typeof(BrainSentimentIndicatorUniverse),
+                Symbol.Create("AAPL", SecurityType.Base, Market.USA, baseDataType: typeof(BrainSentimentIndicatorUniverse)),
+                Resolution.Daily,
+                TimeZones.Utc,
+                TimeZones.Utc,
+                false,
+                false,
+                false,
+                true,
+                null
+            );
+
+            var date = DateTime.Today;
+            var a = factory.Reader(config, line, date, false);
```

Comment:
- `DateTime.Today` uses local time zone, generically should use `DateTime.UtcNow` or even better a specific `DateTime` instance
- Config isn't used, can directly pass in `null`? 
- `a` ?
- `Console.WriteLine(a.ToString());` 99.9% of the cases shouldn't be logging in tests, should directly assert the result is the expected

Same comments for the other added unit test